libs/wlgen/wlgen/rta.py

```diff
@@ -483,32 +498,41 @@ def conf(self,
         workloads. The classes supported so far are detailed hereafter.
 
         Custom workloads
-        ----------------
-        When 'kind' is 'custom' the tasks generated by this workload are the
-        ones defined in a provided rt-app JSON configuration file.
-        In this case the 'params' parameter must be used to specify the
-        complete path of the rt-app JSON configuration file to use.
-
+          When 'kind' is 'custom' the tasks generated by this workload are the
+          ones defined in a provided rt-app JSON configuration file.
+          In this case the 'params' parameter must be used to specify the
+          complete path of the rt-app JSON configuration file to use.
 
         Profile based workloads
-        -----------------------
-        When 'kind' is 'profile' the tasks generated by this workload have a
-        profile which is defined by a sequence of phases and they are defined
-        according to the following grammar:
+          When ``kind`` is "profile", ``params`` is a dictionary mapping task
+          names to task specifications. The easiest way to create these task
+          specifications using :meth:`RTATask.get`.
+
+          For example, the following configures an RTA workload with a single
+          task, named 't1', using the default parameters for a Periodic RTATask:
+
+          ::
+
+            wl = RTA(...)
+            wl.conf(kind='profile', params={'t1': Periodic().get()})
 
-          params := {task, ...}
-          task   := NAME : {SCLASS, PRIO, [phase, ...]}
-          phase  := (PTIME, PRIOD, DCYCLE)
+        :param kind: Either 'custom' or 'profile' - see above.
+        :param params: RT-App parameters - see above.
+        :param duration: Maximum duration of the workload in seconds. Any
+                         remaining tasks are killed by rt-app when this time has
+                         elapsed.
+        :param cpus: CPUs to restrict this workload to, using ``taskset``.
```

Comment:
Is it not possible with sphinx syntax to specify the type of a parameter.
For example in this case it would be useful to tell the user which format has to be used to represent the CPUs mask, e.g. a str or a list(int) ?